src/dedent.ts

```diff
@@ -19,7 +19,10 @@ function createDedent(options: DedentOptions) {
 		...values: unknown[]
 	) {
 		const raw = typeof strings === "string" ? [strings] : strings.raw;
-		const { escapeSpecialCharacters = Array.isArray(strings) } = options;
+		const {
+			escapeSpecialCharacters = Array.isArray(strings),
+			trimWhitespace: stripTrailingWhitespace = true,
```

Comment:
I'm guessing the name switched? :)

```suggestion
			trimWhitespace = true,
```